src/main/kotlin/com/jfrog/conan/clionplugin/services/RemotesDataStateService.kt

```diff
@@ -30,6 +37,19 @@ class RemotesDataStateService : PersistentStateComponent<RemotesDataStateService
     override fun loadState(newState: State) {
         fireStateChangeListener(newState)
         state = newState
+        application.saveSettings()
+    }
+
+    override fun noStateLoaded() {
+        val baseContent = javaClass.classLoader.getResource("conan/base-data.json")?.readText()
```

Comment:
Is always called at startup, so we are not actually persisting the state :( That means that changes to it are not reflected after a restar, even with the `application.saveSettings()` call above (And the saveThreshold disabled for this so it would not wait 5 mins to save things!)

Reply:
If states are not working correctly we may just distribute the plugin with the json and read it and save it each time directly?
Because if not, even doing this, you will always be loading the original json we distributed with the plugin.
WDYT?

Reply:
Yes, let's store it on the user's home folder as you did with the other props